### ISSUE TITLE
bug: args should be optional

### ISSUE THREAD

**Issue by rpocase** [kewisch/jirablueprint#1]
The current implementation forces a user to provide args even if a template may not care about them. While this makes sense for most use cases, if using jirabp just to create a tree with pre-determined similarities, then args likely provides little value. This is trivial to workaround with a stub args template.

```
test:
   args:
      stub:
          description: Satisfying schema
          required: false
```

**Comment by kewisch** [comment]
A template without args seems to work for me, are you getting errors?

```yaml
test:
  issues:
    - fields:
        issuetype: Task
        summary: Test
```

**Comment by kewisch** [comment]
Should be fixed with 485ce7d4ef8994fc0c9c78c7ed08e8609b7872f6
